src/spectrum_assignment.py

```diff
@@ -34,7 +34,7 @@ def _allocate_best_fit(self, channels_list: list):
                     self.spec_help_obj.start_index = start_index
                     self.spec_help_obj.end_index = end_index
                     self.spec_help_obj.core_num = channel_dict['core']
-                    # TODO: This needs to be double checked
+                    # TODO: This needs to be double checked to ensure the correct band is being used
                     self.spec_help_obj.curr_band = channel_dict['band']
```

Comment:
If something needs to be double checked then a test probably needs to be created.